src/ucp/wireup/select.c

```diff
@@ -1431,12 +1549,17 @@ ucs_status_t ucp_wireup_select_aux_transport(ucp_ep_h ep,
                                              unsigned address_count,
                                              ucp_wireup_select_info_t *select_info)
 {
-    ucp_wireup_criteria_t criteria = {0};
+    ucp_wireup_select_ctx_t select_ctx = {0};
+    ucp_wireup_criteria_t criteria     = {0};
+
+    select_ctx.ep            = ep;
+    select_ctx.params        = params;
+    select_ctx.address_list  = address_list;
+    select_ctx.address_count = address_count;
```

Comment:
you can set `select_ctx->allow_am = 0` here, no need for UCP_WIREUP_LANE_USAGE_AUX

Reply:
now USAGE_AUX is needed to check whether it can be emulated over AM or not (for this lane it is always 0, n/e/ can't emulate)